Merge crowdin updates

BacLuc opened the pull request:
That we have a fixed set of changes we merge.

carlobeltrame committed:
New translations email+intl-icu.en.json (English)

carlobeltrame committed:
New translations validators.en.yml (English)

carlobeltrame committed:
Update source file email+intl-icu.en.json

carlobeltrame committed:
Update source file validators.en.yml

carlobeltrame committed:
New translations en.json (French, Switzerland)

carlobeltrame committed:
New translations en.json (German, Switzerland)

carlobeltrame committed:
New translations en.json (German, Switzerland)

carlobeltrame committed:
New translations validators.en.yml (German, Switzerland)

carlobeltrame committed:
New translations email+intl-icu.en.json (English)

carlobeltrame committed:
New translations validators.en.yml (English)

ecampbot committed:
Update source file email+intl-icu.en.json

ecampbot committed:
Update source file validators.en.yml

ecampbot committed:
New translations en.json (German, Switzerland)

ecampbot committed:
New translations en.json (French, Switzerland)

ecampbot committed:
New translations validators.en.yml (English)

ecampbot committed:
Update source file validators.en.yml

ecampbot committed:
New translations email+intl-icu.en.json (English)

ecampbot committed:
New translations email+intl-icu.en.json (German, Switzerland)

ecampbot committed:
New translations validators.en.yml (English)

ecampbot committed:
Update source file email+intl-icu.en.json

ecampbot committed:
Update source file validators.en.yml

ecampbot committed:
New translations en.json (German, Switzerland)

ecampbot committed:
New translations email+intl-icu.en.json (English)

ecampbot committed:
New translations validators.en.yml (English)

ecampbot committed:
New translations validators.en.yml (German, Switzerland)

ecampbot committed:
Update source file validators.en.yml

ecampbot committed:
New translations validators.en.yml (English)

ecampbot committed:
Update source file en.json

ecampbot committed:
Update source file validators.en.yml

ecampbot committed:
New translations en.json (English)

ecampbot committed:
New translations en.json (German, Switzerland)

ecampbot committed:
New translations en.json (French, Switzerland)

ecampbot committed:
New translations validators.en.yml (English)

ecampbot committed:
New translations validators.en.yml (German, Switzerland)

ecampbot committed:
Update source file en.json

ecampbot committed:
Update source file validators.en.yml

carlobeltrame committed:
Adjust tests to changed translations

ecampbot committed:
Update source file en.json

ecampbot committed:
Update source file validators.en.yml

ecampbot committed:
New translations email+intl-icu.en.json (German (Pfadi-spezifische Ausnahmen))

ecampbot committed:
New translations validators.en.yml (German (Pfadi-spezifische Ausnahmen))

ecampbot committed:
Update source file validators.en.yml

ecampbot committed:
New translations validators.en.yml (German, Switzerland)

ecampbot committed:
New translations validators.en.yml (German (Pfadi-spezifische Ausnahmen))

carlobeltrame committed:
Delete unnecessary scout overrides

BacLuc committed:
validators.de_CH.yml: fix test message

BacLuc committed:
Revert "Update source file validators.en.yml"

This reverts commit 543cf5c71636d72dcb51e0824b18183463851641.

manuelmeister reviewed (approved):
I think this would benefit of a squash merge

BacLuc committed:
Merge remote-tracking branch 'upstream/devel' into merge-crowdin-updates

# Conflicts:
#	frontend/src/locales/de.json

BacLuc committed:
frontend: fix grammar error in de.json

Apply the fix from crowdin to dialogActivityCreate
also to dialogCategoryCreate.
